functions-and-scope.js

```diff
@@ -53,14 +75,25 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // averageGrade([6, 4, 5]) geeft xxxx
 // averageGrade([8, 9, 4, 6, 10]) geeft xxxx
 
+function calculateAverage(array) {
+    let total = 0;
+    let count = 0;
+    for (let i = 0; i < array.length; i++) {
+        total += array[i]
+        count++;
+    }
+    return Math.round(total / count)
```

Comment:
`count` hoef je in principe niet bij te houden, omdat die altijd gelijk zal zijn aan `array.length`
```suggestion
    return Math.round(total / array.length)
```